From ce897ac1e02cb26eb2fbe02547d4598269b299c2 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Thu, 2 Nov 2023 17:45:53 +0100
Subject: [PATCH 0017/1695] core: io: fix missing include of delay.h

Adds missing include of kernel/delay.h for udelay() used in
IO_READ32_POLL_TIMEOUT() macro.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git c562219772ca1987a6ff9a0647426d26ec7a00f0 core: io: fix missing include of delay.h]
Change-Id: I856cd2e27237e374e82087f652e5215b703f88f0
Fixes: 97ea199a2ae8 ("core: io: IO_READ32_POLL_TIMEOUT()")
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
[etienne: cherry-picked from 239dffeb3a8b8589585d2b53c5f069d19d872117]
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/include/io.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/include/io.h b/core/include/io.h
index 85c885809..0ef91907b 100644
--- a/core/include/io.h
+++ b/core/include/io.h
@@ -6,6 +6,7 @@
 #define __IO_H
 
 #include <compiler.h>
+#include <kernel/delay.h>
 #include <stdint.h>
 #include <types_ext.h>
 #include <utee_defines.h>
-- 
2.47.2

